src/api/java/com/minecolonies/api/tileentities/TileEntityEnchanter.java

```diff
@@ -0,0 +1,113 @@
+package com.minecolonies.api.tileentities;
+
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.util.ITickable;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.math.MathHelper;
+
+import java.util.Random;
+
+/**
+ * Class which handles the tileEntity of our colonyBuildings.
+ */
+@SuppressWarnings("PMD.ExcessiveImports")
+public class TileEntityEnchanter extends TileEntityColonyBuilding implements ITickable
```

Comment:
Does this TE have to be an ITickable? I’m not seeing any overridden methods from that Interface in here

Reply:
yes, it ticks